src/Spout/Reader/XLSX/Reader.php

```diff
@@ -74,9 +74,9 @@ protected function openReader($filePath)
         if ($this->zip->open($filePath) === true) {
             $this->sharedStringsHelper = new SharedStringsHelper($filePath, $this->getOptions()->getTempFolder());
 
-            if ($this->sharedStringsHelper->hasSharedStrings()) {
+            if ($sharedStringsPath = $this->sharedStringsHelper->getSharedStringsPath()) {
```

Comment:
The shared strings path should remain private. There's no need to expose the path: the path should only be known by the shared strings helper. Therefore, I would keep the `hasSharedStrings()` call here and don't pass anything to the `extractSharedStrings`